src/core/Env.js

```diff
@@ -288,6 +288,13 @@ getJasmineRequireObj().Env = function(j$) {
       return spyRegistry.spyOn.apply(spyRegistry, arguments);
     };
 
+    var ensureIsFunction = function(fn, caller) {
+      var fnType = Object.prototype.toString.call(fn);
```

Comment:
You should be able to just use `j$.isFunction_` here instead of writing it again